### PR TITLE
feature: create a new page movie list

### LINKED ISSUE #17: 5578b212 - create a new page movie list

### PR DESCRIPTION
**Pull request by cesar-ferreira**

### CODE REVIEW

File: movie_badgers/lib/core/routes/manager_route.dart [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+import 'package:moviebadgers/modules/movie_list/view/page/movie_list_page.dart';
+import 'package:moviebadgers/modules/splash/view/page/splash_page.dart';
+
+class ManagerRoute {
+
+  static const SPLASH_PAGE = '/';
+  static const MOVIE_LIST_PAGE = '/movie-list';
```
Comment (samirtf) [inline movie_badgers/lib/core/routes/manager_route.dart]: static const splashPage = '/';
static const movieListPage = '/movie-list';
https://dart.dev/guides/language/effective-dart/style

File: movie_badgers/lib/core/routes/manager_route.dart [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+
+    switch( settings.name ) {
+
+      case SPLASH_PAGE :
+        return MaterialPageRoute(
+            builder: (_) => SplashPage()
+        );
+
+      case MOVIE_LIST_PAGE :
```
Comment (samirtf) [inline movie_badgers/lib/core/routes/manager_route.dart]: case splashPage:
...
case movieListPage:
...
https://dart.dev/guides/language/effective-dart/style

**Review by samirtf — COMMENTED** [review]
We have to discuss about some lint rules.

File: movie_badgers/lib/core/routes/manager_route.dart (hunk 1 above)
Comment (cesar-ferreira) [reply movie_badgers/lib/core/routes/manager_route.dart]: Resolved

File: movie_badgers/lib/core/routes/manager_route.dart (hunk 2 above)
Comment (cesar-ferreira) [reply movie_badgers/lib/core/routes/manager_route.dart]: Resolved

**Review by samirtf — APPROVED** [review]
Thank you for your collaboration.

**Pull request merged**